native/optflow.h

```diff
@@ -0,0 +1,61 @@
+#ifndef CV_RS_OPTFLOW_H
+#define CV_RS_OPTFLOW_H
```

Comment:
@Pzixel could the cc files use `#pragma once`? I think basically any compiler that supports opencv would support that pragma

Reply:
All other files do use "traditional" `#ifdef` gating - probably not a good idea to be inconsistent for this file only.

I suggest we leave it as it is and *then* change all files in another PR *afterwards* if necessary

```
nico@devuan-3xS:~/Projects/3rdParty/cv-rs/native$ grep CV_RS *.h
common.h:#ifndef CV_RS_COMMON_H
common.h:#define CV_RS_COMMON_H
common.h:#endif  // CV_RS_COMMON_H
common-rust.h:#ifndef CV_RS_COMMON_RUST_H
common-rust.h:#define CV_RS_COMMON_RUST_H
common-rust.h:#endif  // CV_RS_COMMON_RUST_H
features2d.h:#ifndef CV_RS_FEATURES2D_H
features2d.h:#define CV_RS_FEATURES2D_H
features2d.h:#endif  // CV_RS_FEATURES2D_H
highgui.h:#ifndef CV_RS_HIGHGUI_H
highgui.h:#define CV_RS_HIGHGUI_H
highgui.h:#endif  // CV_RS_HIGHGUI_H
imcodecs.h:#ifndef CV_RS_IMCODECS_H
imcodecs.h:#define CV_RS_IMCODECS_H
imcodecs.h:#endif  // CV_RS_IMCODECS_H
imgproc.h:#ifndef CV_RS_IMGPROC_H
imgproc.h:#define CV_RS_IMGPROC_H
imgproc.h:#endif  // CV_RS_IMGPROC_H
mat.h:#ifndef CV_RS_MAT_H
mat.h:#define CV_RS_MAT_H
mat.h:#endif  // CV_RS_MAT_H
objdetect.h:#ifndef CV_RS_OBJDETECT_H
objdetect.h:#define CV_RS_OBJDETECT_H
objdetect.h:#endif  // CV_RS_OBJDETECT_H
optflow.h:#ifndef CV_RS_OPTFLOW_H
optflow.h:#define CV_RS_OPTFLOW_H
text.h:#ifndef CV_RS_TEXT_H
text.h:#define CV_RS_TEXT_H
text.h:#endif  // CV_RS_TEXT_H
video.h:#ifndef CV_RS_VIDEO_H
video.h:#define CV_RS_VIDEO_H
video.h:#endif  // CV_RS_VIDEO_H
videoio.h:#ifndef CV_RS_VIDEOIO_H
videoio.h:#define CV_RS_VIDEOIO_H
videoio.h:#endif  // CV_RS_VIDEOIO_H
```